styles.css

```diff
@@ -104,9 +104,9 @@ arranged in order that they appear in the html
 
 .gameplay-cont {
   padding: 10px;
-  width: 440px;
   background-color: white;
   border: solid 2px rgb(238,238,238);
+  margin-top: 15px;
```

Comment:
can we make this margin to a percentage so it scales? i had put 2.5% on top but am open to a different value